src/api/api.ts

```diff
@@ -44,6 +48,7 @@ export function Api<T>(
       return {
         message,
         stat: 'fail',
+        error: err.response.data.error ?? null,
```

Comment:
We could have added this in the if else case above to a local variable. But if you want to keep it here then shouldnt there be null check for data within err.response?

Reply:
Yes, I missed it before and i have added it now.